app/models/mixins/scanning_mixin.rb

```diff
@@ -368,15 +368,32 @@ def sync_stashed_metadata(ost)
       save_metadata_op(xml_summary.miqEncode, "b64,zlib,xml", ost.taskid)
       _log.info "Completed: Sending target summary to server.  TaskId:[#{ost.taskid}]  target:[#{name}]"
 
-      update_agent_message(ost, "Synchronization complete")
+      update_job_message(ost, "Synchronization complete")
 
       raise syncErr if syncErr
     end
     ost.value = "OK\n"
   end
 
-  def update_agent_message(ost, message)
-    ost.agent_message = message
-    agent_job_message_op(ost.taskid, ost.agent_message) if ost.taskid && ost.taskid.empty? == false
+  def update_job_message(ost, message)
+    ost.message = message
+    unless ost.taskid.blank?
+      MiqQueue.put(
+        :class_name  => "Job",
+        :method_name => "update_message",
+        :args        => [ost.taskid, message],
+        :task_id     => "job_message_#{Time.now.to_i}",
+        :zone        => MiqServer.my_zone,
+        :role        => "smartstate"
+      )
+      MiqQueue.put(
+        :class_name  => "Job",
+        :method_name => "update_worker_heartbeat",
+        :args        => [ost.taskid],
+        :task_id     => "job_worker_heartbeat#{Time.now.to_i}",
+        :zone        => MiqServer.my_zone,
+        :role        => "smartstate"
+      )
+    end
```

Comment:
Not sure why this is 2 queue items, when it was only one previously.

Note that https://github.com/ManageIQ/manageiq/pull/14767/files?diff=split#diff-96955d17ecd8f0296297c7dc1392d347R88, even though it's run through MiqQueue class, does not actually run on the queue asynchronously.